tmt/hardware.py

```diff
@@ -904,11 +942,28 @@ def wrapper(spec: Spec, index: int) -> BaseConstraint:
     return wrapper
 
 
-def _parse_number_constraints(
+def _parse_int_constraints(
+        spec: Spec,
+        prefix: str,
+        constraint_keys: tuple[str, ...]) -> list[BaseConstraint]:
+    """ Parse number-like constraints defined by a given set of keys, to int """
+
+    return [
+        IntegerConstraint.from_specification(
+            f'{prefix}.{constraint_name.replace("-", "_")}',
+            str(spec[constraint_name]),
+            allowed_operators=[
+                Operator.EQ, Operator.NEQ, Operator.LT, Operator.LTE, Operator.GT, Operator.GTE])
+        for constraint_name in constraint_keys
+        if constraint_name in spec
+        ]
+
+
+def _parse_float_constraints(
```

Comment:
Should be `_parse_number_constraints()`, to match the `NumberConstraint` class name.